R/rent_assistance.R

```diff
@@ -15,6 +15,11 @@
 #' @param max_rate If not \code{NULL}, a numeric vector indicating for each individual the maximum rent assistance payable.
 #' @param min_rent If not \code{NULL}, a numeric vector indicating for each individual the minimum fortnightly rent above which rent assistance is payable. \code{max_rate} and \code{min_rent} must not be used when \code{fy.year} is set.
 #' 
+#' @param sharers_provision_applies (logical, default: FALSE) Does the sharers provision apply to the parent payment? The list of functions can be found in table 2 column 4 http://guides.dss.gov.au/guide-social-security-law/3/8/1/10
```

Comment:
Added the link to the relevant guide, not sure if his is best practice but thought it would be useful for future references